base/src/main/java/uk/ac/ebi/atlas/species/AtlasInformationDao.java

```diff
@@ -0,0 +1,44 @@
+package uk.ac.ebi.atlas.species;
+
+import com.google.gson.stream.JsonReader;
+import uk.ac.ebi.atlas.model.resource.AtlasResource;
+import uk.ac.ebi.atlas.model.resource.JsonFile;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+@Named
+public class AtlasInformationDao {
+
+    private final AtlasResource<JsonReader> atlasInformationJsonFile;
+
+    @Inject
+    public AtlasInformationDao(Path atlasInformationFilePath) {
+        atlasInformationJsonFile =
+                new JsonFile.ReadOnly(
+                        atlasInformationFilePath.getParent().toString(),
+                        atlasInformationFilePath.getFileName().toString());
+    }
+
+    public ArrayList<String> fetchAll() throws IOException {
```

Comment:
I know you modelled this on `SpeciesPropertiesDao`, but I think the JSON parsing logic could be simplified a LOT if you used [Gson](https://google.github.io/gson/apidocs/com/google/gson/Gson.html#fromJson-java.io.Reader-java.lang.Class-). So your code could be as simple as:
```
 JsonReader reader = new JsonReader(new FileReader(atlasInformationFilePath.toString()));
 SomeClass data = GsonProvider.GSON.fromJson(reader, SomeCLass.class);
```

@alfonsomunozpomer Is there are a reason why the `SpeciesPropertiesDao` is so convoluted?